manim/mobject/mobject.py

```diff
@@ -2149,10 +2153,11 @@ def get_nadir(self) -> Point3D:
 
     def length_over_dim(self, dim: int) -> float:
```

Comment:
As this function is meant to be used only internally, it should be private
```suggestion
    def __length_over_dim(self, dim: int) -> float:
```
or at least protected
```suggestion
    def _length_over_dim(self, dim: int) -> float:
```

Reply:
It makes sense. In that case, `.reduce_across_dimension()` is an even stronger case of a method which should be marked as internal.

There's already a PR which proposes marking some methods as "internal". It's worth checking it:
https://github.com/ManimCommunity/manim/pull/3793